mllib/src/test/scala/org/apache/spark/ml/clustering/GaussianMixtureSuite.scala

```diff
@@ -268,6 +268,13 @@ class GaussianMixtureSuite extends MLTest with DefaultReadWriteTest {
     assert(trueLikelihood ~== doubleLikelihood absTol 1e-6)
     assert(trueLikelihood ~== floatLikelihood absTol 1e-6)
   }
+
+  test("prediction on single instance") {
+    val gmm = new GaussianMixture()
```

Comment:
maybe explicitly set seed in order to minimize randomness from run to run (usually better to make tests as deterministic as possible)